src/components/ToastNotifications.js

```diff
@@ -7,10 +7,12 @@ import { setNotificationNotified } from '_/actions'
 
 import style from './sharedStyle.css'
 
+const notificationTypes = ['error', 'warning', 'success', 'info', 'danger']
 const ToastNotifications = ({ userMessages, onDismissNotification }) => {
   return <ToastNotificationList>
+
     { userMessages.get('records').filter(r => !r.get('notified')).map(r =>
-      <TimedToastNotification className={style['toast-margin-top']} type='warning' onDismiss={() => onDismissNotification(r.get('time'))} key={r.get('time')}>
+      <TimedToastNotification className={style['toast-margin-top']} type={notificationTypes.includes(r.get('type')) ? `${r.get('type')}` : 'warning'} onDismiss={() => onDismissNotification(r.get('time'))} key={r.get('time')}>
```

Comment:
- This is a case sensitive check ... maybe `notificationTypes.includes(r.get('type').toLowerCase()`
 - The code `` `${r.get('type')}` `` can be reduced to just `r.get('type')`.